src/models/tiles/question/question-content.ts

```diff
@@ -1,31 +1,51 @@
-import stringify from "json-stringify-pretty-compact";
-import { types, Instance, SnapshotIn, getSnapshot } from "mobx-state-tree";
+import { types, Instance, SnapshotIn } from "mobx-state-tree";
 import { ITileContentModel, TileContentModel } from "../tile-content";
-import { ITileExportOptions } from "../tile-content-info";
+import { ITileExportOptions, IDefaultContentOptions } from "../tile-content-info";
+import { RowList } from "../../document/row-list";
+import { kPlaceholderTileType } from "../placeholder/placeholder-content";
+import { StringBuilder } from "../../../utilities/string-builder";
+import { ITileModel } from "../tile-model";
 
 export const kQuestionTileType = "Question";
 
-export function defaultQuestionContent() {
-  return QuestionContentModel.create();
+export function defaultQuestionContent(options?: IDefaultContentOptions) {
+  // Create a placeholder tile
+  const placeholderTile = options?.tileFactory?.(kPlaceholderTileType);
+  if (!placeholderTile) {
+    throw new Error("Placeholder tile could not be created");
+  }
+  const tile = QuestionContentModel.create({});
+  tile.addRowWithTiles([placeholderTile]);
+
+  return tile;
 }
 
-export const QuestionContentModel = TileContentModel
-  .named("QuestionContent")
+export const QuestionContentModel = types.compose(
+    "QuestionContent",
+    TileContentModel,
+    RowList)
   .props({
     type: types.optional(types.literal(kQuestionTileType), kQuestionTileType),
     version: types.optional(types.number, 1),
-    locked: types.optional(types.boolean, false)
+    locked: types.optional(types.boolean, false),
   })
   .views(self => ({
-    exportJson(options?: ITileExportOptions) {
-      const snapshot = getSnapshot(self);
-      return stringify(snapshot, {maxLength: 200});
+    exportJson(options: ITileExportOptions, tileMap: Map<string, ITileModel>) {
```

Comment:
Its too bad that we are still having to support this. Most of the time just the "raw" MST serialization is used. And I think this is really just used for authoring to make it more human readable. However I think it is less and less common that Leslie looks at the actual JSON anymore.